primitives/trie/src/lib.rs

```diff
@@ -532,6 +536,17 @@ mod tests {
 		assert_eq!(root1, root2);
 	}
 
+
+	#[test]
+	fn delta_trie_root_works_with_empty_db() {
+		let mut db = MemoryDB::default();
+		delta_trie_root::<Layout, _, _, _, _, _>(
+			&mut db,
+			Default::default(),
```

Comment:
could also be 
`empty_child_trie_root::<Layout>(),` (yes name of the function is not good but it gives the default empty chid trie root that is the root of a empty trie).

Reply:
This is not about child tries here?